README.md

```diff
@@ -18,6 +18,22 @@ curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-
 (Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py -UseBasicParsing).Content | python -
 ```
 
+## Deploying with Ansible
+
+This project is setup to be able to be deployed using Ansible. To deploy, run the following:
+
+```bash 
+$ ansible-playbook playbook.yaml -i inventory.yaml
+```
+This will prompt you for three values:
+* Flask secret key
+* Trello token
+* Trello API key
+
+Inputting these values will allow ansible to deploy the app. Board IDs are currently hardcoded for convenience into the Jinja template as these values are non-sensitive. 
+
+You should now be able to view the hosted app at http://3.11.233.49:5000
```

Comment:
Thanks for the clear documentation!

Note that we _won't_ be using this again, so there's no need to keep this section (or the playbook etc.) up to date unlike most exercises